app/assets/stylesheets/application.css

```diff
@@ -17,7 +17,19 @@
 
 
 body {
-    background-color: #f8e8ee; 
+    background-color: #f8e8ee;
 }
+.simple-calendar {
+    width: 100%;
+    border: 1px solid #ccc;
+    font-size: 12px;
+    font-family: Arial, Helvetica, sans-serif;
+    color: #e0868f;
+    background-color: #fff;
+  }
+.table td {
+  width: 14%;
+}
```

Comment:
This style will be applied to all tables in your entire system, I don't think that's what you want. 
Please add either an ID or a class to your calendar table and apply this style to it only.

Reply:
I added the day element to my table, it seemed easier to me because I don't have tables in my view made by me, it works and it doesn't affect my other table, I'm just not sure it's the best option